lib/lz4.c

```diff
@@ -124,9 +125,11 @@
 /**************************************
 *  Memory routines
 **************************************/
+#if (HEAPMODE)
```

Comment:
<stdlib.h> can be moved outside of #if. If some function in project use malloc or free in debug build, compiler include all heap management stuff including special segment for heap etc.. In this case I get linking warnings.